avatar_badge.css

```diff
@@ -0,0 +1,95 @@
+.avatar{
+    border-radius: 50%;
+  }
+  
+  .square-avatar{
+    border-radius: 0.2rem;
+  }
+  
+  .avatar-lg{
+    height: 14rem;
+    width: 14rem;
+  }
+  
+  .avatar-md{
+    height: 9rem;
+    width: 9rem;
+  }
+  
+  .avatar-sm{
+    height: 6rem;
+    width: 6rem;
+  }
+  
+  
+  /* badge */
+  
+  .badge{
+    position: relative;
+  }
+  
+  .online{
+    height: 2rem;
+    width: 2rem;
+    border-radius: 50%;
+    background-color: green;
+    position: absolute;
+    right: 0;
+    bottom: 8%;
+    border: 3px solid white;
+  }
+  
+  .offline{
```

Comment:
for  .online and .offline class you are repeating the same code for changing the background color just add a new class

Reply:
done